includes/wizards/class-wizard.php

```diff
@@ -54,11 +54,21 @@ abstract class Wizard {
 	public $parent_menu = '';
 
 	/**
-	 * Order relative to the Newspack Dashboard menu item.
+	 * Parent menu order relative to the Newspack Dashboard menu item.
 	 *
 	 * @var int
 	 */
-	public $menu_order = 0;
+	public $parent_menu_order = 0;
+
+	/**
+	 * Priority for when 'add_page' callback fires. Default is WordPress's default hook priority of 10
+	 * Override this on a per-wizard basis if there is a need to adjust the order that 'add_page' fires.
+	 * Example: set priority < 10 so add_page is called before other pages. Or > 10 so it's called after other
+	 * pages or plugins are loaded.
+	 *
+	 * @var int.
+	 */
+	protected $add_page_priority = 10;
```

Comment:
I remember we thought of `submenu_priority` here, but I like this idea better. Following the same logic, the priority is applied to the `admin_menu` hook, so perhaps it should be called `admin_menu_priority`. WDYT? This is non-blocking, though.